.eslintcache

```diff
@@ -1 +1 @@
-[{"Z:\\Dev\\project-hunt\\src\\App.js":"1","Z:\\Dev\\project-hunt\\src\\index.js":"2","Z:\\Dev\\project-hunt\\src\\components\\Navbar.js":"3","Z:\\Dev\\project-hunt\\src\\components\\Layout.js":"4","Z:\\Dev\\project-hunt\\src\\pages\\HomePage.js":"5","Z:\\Dev\\project-hunt\\src\\pages\\ProjectShowcase.js":"6","Z:\\Dev\\project-hunt\\src\\pages\\Login.js":"7","Z:\\Dev\\project-hunt\\src\\reportWebVitals.js":"8"},{"size":666,"mtime":1606222685818,"results":"9","hashOfConfig":"10"},{"size":494,"mtime":1606225007880,"results":"11","hashOfConfig":"10"},{"size":906,"mtime":1606229947930,"results":"12","hashOfConfig":"10"},{"size":224,"mtime":1606230002090,"results":"13","hashOfConfig":"10"},{"size":213,"mtime":1606230014773,"results":"14","hashOfConfig":"10"},{"size":234,"mtime":1606230024995,"results":"15","hashOfConfig":"10"},{"size":204,"mtime":1606230021575,"results":"16","hashOfConfig":"10"},{"size":375,"mtime":1606213533290,"results":"17","hashOfConfig":"10"},{"filePath":"18","messages":"19","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"yqetui",{"filePath":"20","messages":"21","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"22"},{"filePath":"23","messages":"24","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"25","messages":"26","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"27","messages":"28","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"29","messages":"30","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"31","messages":"32","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"33","messages":"34","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"Z:\\Dev\\project-hunt\\src\\App.js",[],"Z:\\Dev\\project-hunt\\src\\index.js",[],["35","36"],"Z:\\Dev\\project-hunt\\src\\components\\Navbar.js",[],"Z:\\Dev\\project-hunt\\src\\components\\Layout.js",[],"Z:\\Dev\\project-hunt\\src\\pages\\HomePage.js",[],"Z:\\Dev\\project-hunt\\src\\pages\\ProjectShowcase.js",[],"Z:\\Dev\\project-hunt\\src\\pages\\Login.js",[],"Z:\\Dev\\project-hunt\\src\\reportWebVitals.js",[],{"ruleId":"37","replacedBy":"38"},{"ruleId":"39","replacedBy":"40"},"no-native-reassign",["41"],"no-negated-in-lhs",["42"],"no-global-assign","no-unsafe-negation"]
```

Comment:
Don't commit this file. it can be added to `.gitignore` so that it's ignored by git.